neofs/neofs_contract.go

```diff
@@ -253,6 +252,12 @@ func OnNEP17Payment(from interop.Hash160, amount int, data interface{}) {
 		return
 	}
 
+	if amount <= 0 {
+		panic("onNEP17Payment: amount must be positive")
+	} else if maxBalanceAmount < amount {
```

Comment:
Update `maxBalanceAmount` constant. It should be 10^8 times bigger value.

Reply:
Fixed.